storage-service/src/main/java/io/quarkus/qe/consumers/UpdateRepositoryRequestConsumer.java

```diff
@@ -71,4 +78,19 @@ private void updateLogs(Repository repository, RepositoryEntity entity) {
             }
         }
     }
+
+    private void updateVersion(Repository repository, RepositoryEntity entity) {
+        Optional.ofNullable(repository.getGlobalVersion()).ifPresent(version -> {
+            PanacheQuery<QuarkusVersionEntity> persistedVersionEntity = QuarkusVersionEntity.find("version",
```

Comment:
If the version would be the primary key, this would be considerably easier to do: just QuarkusVersionEntity.findById(xx).